data/json/itemgroups/science_and_tech.json

```
@@ -13,6 +13,7 @@
       { "item": "chem_washing_soda", "prob": 15, "charges-min": 1 },
       { "item": "chem_baking_soda", "prob": 5, "charges-min": 1 },
       [ "mutagen", 8 ],
+      [ "mutagen_chelator", 15 ],
       [ "purifier", 12 ],
```

```suggestion
      [ "mutagen_chelator", 7 ],
      [ "purifier", 5 ],
```
So the total amount of purifier+chelator is the same as purifier currently, and chelator is a little more common.